app/uk/gov/hmrc/incometaxproperty/connectors/IFConnector.scala

```diff
@@ -28,6 +28,8 @@ trait IFConnector {
 
   protected val appConfig: AppConfig
 
+  protected[connectors] lazy val baseUrl: String = if (appConfig.ifEnvironment == "test") appConfig.ifBaseUrl + "/if" else appConfig.ifBaseUrl
```

Comment:
we don't need to add this `/if` anywhere? This is only for the stub

Reply:
Removed